app/controllers/taxonomy_todos_controller.rb

```diff
@@ -0,0 +1,27 @@
+class TaxonomyTodosController < ApplicationController
+  def redirect_to_first
+    taxonomy_project = TaxonomyProject.find(params[:taxonomy_project_id])
+    redirect_to taxonomy_project.taxonomy_todos.still_todo.first
+  end
+
+  def show
+    taxonomy_todo = TaxonomyTodo.find(params[:id])
+    @todo_form = TaxonomyTodoForm.new(taxonomy_todo: taxonomy_todo)
+  end
+
+  def update
+    taxonomy_todo = TaxonomyTodo.find(params[:id])
+    todo_form = TaxonomyTodoForm.new(params.require(:taxonomy_todo_form).permit(:new_terms))
+    todo_form.taxonomy_todo = taxonomy_todo
+    todo_form.user = current_user
+    todo_form.save
+
+    next_item = taxonomy_todo.taxonomy_project.taxonomy_todos.still_todo.first
```

Comment:
this feels like it should be a method on the todo model. `TaxonomyTodo#next` or somesuch.